src/gui/passkeys/PasskeyImporter.h

```diff
@@ -33,17 +33,20 @@ class PasskeyImporter : public QObject
     explicit PasskeyImporter() = default;
 
     void importPasskey(QSharedPointer<Database>& database, Entry* entry = nullptr);
-
-private:
-    void importSelectedFile(QFile& file, QSharedPointer<Database>& database, Entry* entry);
-    void showImportDialog(QSharedPointer<Database>& database,
+    bool showImportDialog(QSharedPointer<Database>& database,
                           const QString& url,
                           const QString& relyingParty,
                           const QString& username,
                           const QString& credentialId,
                           const QString& userHandle,
                           const QString& privateKey,
-                          Entry* entry);
+                          Entry* entry,
```

Comment:
I prefer substantive parameters like this be at front of list if not defaulting to nullptr. In this case, entry should come right after database.